CONTRIBUTORS.txt

```diff
@@ -17,3 +17,5 @@ Oleh Kuchuk
 Sviatoslav Sydorenko
 Vasyl Navrotsky
 Vladyslav Ovchynnykov
+Alexey Dyatlov
```

Comment:
this line repeats twice `Alexey Dyatlov`

Reply:
Oh, I've missed that.